main/tests/test_views.py

```diff
@@ -49,6 +49,9 @@ def test_index_view_rendering(self):
         response = self.client.get(reverse('post_list'))
         self.assertEqual(200, response.status_code)
         self.assertTemplateUsed(response, 'main/index.html')
+        self.assertContains(response, post)
```

Comment:
No, don't check for model data.

And this check is incorrect as it depends on current time.